upup/pkg/fi/nodeup/command.go

```diff
@@ -244,62 +242,23 @@ func (c *NodeUpCommand) Run(out io.Writer) error {
 	loader.Builders = append(loader.Builders, &model.PackagesBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.SecretBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.FirewallBuilder{NodeupModelContext: modelContext})
-	loader.Builders = append(loader.Builders, &model.NetworkBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.SysctlBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.KubeAPIServerBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.KubeControllerManagerBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.KubeSchedulerBuilder{NodeupModelContext: modelContext})
 	loader.Builders = append(loader.Builders, &model.EtcdManagerTLSBuilder{NodeupModelContext: modelContext})
+	loader.Builders = append(loader.Builders, &model.KubeProxyBuilder{NodeupModelContext: modelContext})
+	loader.Builders = append(loader.Builders, &model.EtcdTLSBuilder{NodeupModelContext: modelContext})
```

Comment:
`EtcdTLSBuilder` needs to be changed to not act when networking is not Calico.

Reply:
Perhaps rename it to `CalicoBuilder`.

Reply:
Good catch. Will look into this

Reply:
This one is a bit misleading. It creates certs named calico, but it is used by any provider that accesses legacy etcd with TLS enabled. For Romana, this seems to be broken, for cilium, I have removed support for this. Calico _could_ probably work if the user works really hard to to configure this. Would have to use calico V2 ++. I think we can just leave this as is and rather work on deprecating legacy etcd further.

Reply:
I do not want to start creating that cert when it previously was not. The appending of `EtcdTLSBuilder` was previously conditional on networking being set to Calico. So either the appending should remain conditional or the builder should only act for Calico.

Reply:
Agreed.

Reply:
Comment resolved by subsequent commit.